health-services/libraries/health-services-models/src/main/java/org/egov/common/models/project/useraction/UserAction.java

```diff
@@ -0,0 +1,108 @@
+package org.egov.common.models.project.useraction;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.egov.common.models.core.EgovOfflineModel;
+import org.egov.common.models.project.TaskAction;
+import org.springframework.validation.annotation.Validated;
+
+/**
+ * The UserAction class represents a user action related to a project.
```

Comment:
The UserAction class represents an action performed by the logged in user related to a project with or without in relation to a beneficiary

Reply:
done